### ISSUE TITLE
ValueError: Clean tolerance is too high when TPMS gyroid thickness is greater than 0.4

### ISSUE THREAD

**Issue by ricardo0115** [3MAH/microgen#9]
Hello, I'm trying to generate a TPMS Gyroid with a thickness greater than 0.4 using the following script 

```python
import microgen

# Generate gyroid

geometry = microgen.Tpms(
    surface_function=microgen.tpms.gyroid,
    type_part="sheet",
    thickness=0.5,
)
shape = geometry.generate()
```

But i have this error : 

```
/home/mguevara/anaconda3/envs/microgen-dev/lib/python3.10/site-packages/pyvista/core/grid.py:508: PyVistaDeprecationWarning: `dims` argument is deprecated. Please use `dimensions`.
  warnings.warn(
2023-02-17 16:52:15.656 (   0.524s) [        738BB740]vtkSmoothPolyDataFilter:327    ERR| vtkSmoothPolyDataFilter (0x55b1dd299b00): No data to smooth!
ERROR:root:No data to smooth!
Traceback (most recent call last):
  File "/tmp/issue_micro.py", line 10, in <module>
    shape = geometry.generate(nSample=30, smoothing=100)
  File "/home/mguevara/anaconda3/envs/microgen-dev/lib/python3.10/site-packages/microgen-1.0.1-py3.10.egg/microgen/shape/tpms.py", line 294, in generate
  File "/home/mguevara/anaconda3/envs/microgen-dev/lib/python3.10/site-packages/microgen-1.0.1-py3.10.egg/microgen/shape/tpms.py", line 181, in createSurfaces
  File "/home/mguevara/anaconda3/envs/microgen-dev/lib/python3.10/site-packages/pyvista/core/filters/poly_data.py", line 1847, in clean
    raise ValueError('Clean tolerance is too high. Empty mesh returned.')
ValueError: Clean tolerance is too high. Empty mesh returned.

```

**Comment by chemiskyy** [comment]
I think we can close this issue considering the new capabilities from #14, especially using generateVTK
The classical generate that uses Cadquery has still to be used with caution, when the two offset surfaces are too close

@ricardo0115 maybe we should add a warning in this case, since we can estimate the offset to create an empty or a full box

**Comment by ricardo0115** [comment]
Solved in #14 using generateVTK, if cadquery method is imperative to TPMS generation, `resolution` argument in `generate` method must be increased.
